packages/core/src/locales.ts

```diff
@@ -1,5 +1,6 @@
 import az from "./locales/az.js";
 import en from "./locales/en.js";
 import es from "./locales/es.js";
+import ta from "./locales/ta.js";
 
-export { az, es, en };
+export { az, es, en, ta };
```

Comment:
This can be probably written as following 

```js
export { default as az } from "./locales/az.js";
export { default as en } from "./locales/en.js";
export { default as es } from "./locales/es.js";
export { default as ta } from "./locales/ta.js";
```

This way, upon merging one locale PR, another locale PR doesn't need to resolve conflict on the `export` before mering.

@colinhacks Please share your thoughts on this.

Reply:
Would've been smart...I've been resolving these conflicts for an hour now 😅